lib/udap_security_test_kit/endpoints/mock_udap_server/udap_response_creation.rb

```diff
@@ -0,0 +1,294 @@
+require_relative '../../urls'
+require_relative '../../tags'
+require_relative '../mock_udap_server'
+require_relative '../../client_suite/oidc_jwks'
+
+module UDAPSecurityTestKit
+  module MockUDAPServer
+    module UDAPResponseCreation
```

Comment:
As in the SMART App Launch PR, this file seems to be doing too much. I don't see why one file should handle the responses for all of the endpoints.

Reply:
I separated out files into token, authorization, and registration. The bulk are in token bit. As with SMART, this is needed to support a combo of SMART and UDAP options in client test kits like PAS and [CARIN](https://github.com/inferno-framework/carin-for-blue-button-test-kit/blob/client_auth/lib/carin_for_blue_button_test_kit/client/v2.0.0/endpoints/token_endpoint.rb#L11). Separating the response generation logic into modules allows them to both be mixed in.